/**/riptide_slam:
  ros__parameters:

    pose_key: "X"
    velocity_key: "V"

    # How long should factors exist before being
    # marginalized out. (In Seconds)
    # NOTE: Must be a decimal
    smoother_lag: 60.0

    plugins:
      imu:
        bias_key: "B"

        imu_list:
          vectornav:
            topic: "topic1"
          zed:
            topic: "topic2"
